app/models.py

```diff
@@ -1351,7 +1351,8 @@ def check_code(self, cde):
 
 FIRST_CLASS = 'first'
 SECOND_CLASS = 'second'
-POSTAGE_TYPES = [FIRST_CLASS, SECOND_CLASS]
+INTERNATIONAL = 'international'
+POSTAGE_TYPES = [FIRST_CLASS, SECOND_CLASS, INTERNATIONAL]
```

Comment:
We need Europe and Non-Europe, instead of international. 
And we'll need to update `RESOLVE_POSTAGE_FOR_FILE_NAME={
 FIRST_CLASS:1, SECOND_CLASS: 2, EUROPE: 'E', NON_EUROPE:'N'
}`

Reply:
I went with `Europe` and `rest of world` here: https://github.com/alphagov/notifications-utils/pull/703/files#diff-0cdb242f4e660d4fe27f8b6bba754b52R60-R63

Would be nice to be consistent.

Reply:
I didn't realise we already had names for these in utils - I've updated this to use the same names